### PR TITLE
Fix running tests on Travis

### PR DESCRIPTION
**Pull request by pali**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: .travis.yml [hunk 1]
```diff
@@ -312,12 +332,25 @@ matrix:
...
       env: DB=MariaDB VERSION=10.3.12
+    - perl: "5.24"
+      env: DB=MariaDB VERSION=10.3.17
     - perl: "5.24"
       env: DB=MariaDB VERSION=10.4.0
     - perl: "5.24"
       env: DB=MariaDB VERSION=10.4.1
     - perl: "5.24"
       env: DB=MariaDB VERSION=10.4.2
+# Incompatible, unsupported by MySQL::Sanbox
```
Comment (choroba) [inline .travis.yml]: San*d*box

**Pull request merged**